src/inmanta/deploy/scheduler.py

```diff
@@ -449,7 +449,10 @@ async def _initialize(self) -> None:
                 # Set running flag because we're ready to start accepting tasks.
                 # Set before scheduling first tasks because many methods (e.g. read_version) skip silently when not running
                 self._running = True
+                # All resources get a timer
                 await self.read_version(connection=con)
+                self._timer_manager.update_timers(restored_state.resources.keys() - self._state.dirty)
```

Comment:
We should probably do this under lock though:
```suggestion
                with self._scheduler_lock:
                    self._timer_manager.update_timers(self._state.resources.keys() - self._state.dirty)
```